packages/react-core/src/demos/SearchInput/SearchInput.md

```diff
@@ -492,7 +492,7 @@ AdvancedComposableSearchInput = () => {
                       id="datePicker"
                       style={{ width: '100%' }}
                       value={date}
-                      onChange={setDate}
+                      onChange={(e,_value) => setDate(_value)}
```

Comment:
Small swap - looks like `e` is the unused var here
```suggestion
                      onChange={(_e, value) => setDate(value)}
```

Reply:
Do we use underscore and name as unused variables? I used it for value as `value` is already defined in the scope

Reply:
Yes the PF convention is `_varName` for unused variables

Reply:
ok, changed the names